src/components/Notification.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <!-- content wrapper  -->
+  <div
+    class="flex flex-row h-24 w-full mx-auto bg-white shadow-md overflow-hidden m-5"
```

Comment:
we don't use shadows anywhere else so to keep it consistent lets add them everywhere else or delete it from here, also I think we could use some more padding cause now the text is super close to the border of the component